content/en/blog/2025-04-10-updates-to-the-open-source-platform-cozystack-0-24-0-29.md

```
@@ -0,0 +1,121 @@
+slug: updates-to-the-open-source-platform-cozystack-0-24-0-29-
+date: 2025-04-10
+author: "Timur Tukaev"
+description: "We haven’t shared much about Cozystack’s new features lately, even though we’ve released six new versions over the past month and a half…"
+---
+
+### Updates to the Open-Source Platform Cozystack 0.24–0.29: PXE Machine Provisioning, Inter-Datacenter RTT Monitoring, and Dedicated IP Addresses for VMs
+
+We haven’t shared much about Cozystack’s new features lately, even though we’ve released six new versions over the past month and a half: 0.24, 0.25, 0.26, 0.27, 0.28, and 0.29. Let’s take a closer look at the changes, starting from the latest release and going back to version 0.24.
+
+> What is Cozystack?
+
```

markdownlint-cli2 (0.18.1)

Blank line inside blockquote

(MD028, no-blanks-blockquote)